data/orders.ddl

```diff
@@ -0,0 +1,6 @@
+create table orders(
+    id INT auto_increment,
+    user_id INT,
+    primary key(id),
```

Comment:
There should be more fields here . ..   maybe order cost, tax, etc?  Look at an order of yours from somewhere